### ISSUE TITLE
Removes translations with emoji

### ISSUE THREAD

**Issue by briankracoff** [Flinesoft/BartyCrouch#7]
I have a UILabel in a storyboard with `😀` as the text. I added a new locale through Xcode which gives me in the .strings file:

```
/* Class = "UILabel"; text = "😀"; ObjectID = "hi9-mx-Y1T"; */
"hi9-mx-Y1T.text" = "😀";
```

However, when I run BartyCrouch, it removes those two lines.

**Comment by Dschee** [comment]
Interesting, let me investigate this a bit ...

**Comment by Dschee** [comment]
@briankracoff Thanks for reporting this, I could reproduce the issue and created a test for it so it is covered for future versions. But I couldn't find the appropriate solution in the short time I have right now.

I created a branch named `work/emoji` where I added the test and a **Playground** called `UnicodeRegexDemo.playground` where you can see the problem: The `NSRegularExpression` which BartyCrouch is using to match the comment and key-value lines in `.strings` files generated by `ibtool` does not work for Unicode valued values, not even values mixed with Unicode symbols.

I'll have a look at it later but in case you have some time, feel free to investigate the issue using the Playground as your starting point.

**Comment by briankracoff** [comment]
@Dschee thanks for taking a look! I think you should be checking `utf16.count` instead of `characters.count` when you make your `NSRange` in  the `matchesInString` calls. I think that should do the trick.

**Comment by Dschee** [comment]
That makes sense, thank you @briankracoff! I just released a new version with the fix (v1.4.1) – all unicode characters including your smiley should work now. :tada:
